src/core/load-fragment.c

```diff
@@ -2744,6 +2744,13 @@ int config_parse_environ(
                 return 0;
         }
 
+        const Specifier table[] = {
+                COMMON_SYSTEM_SPECIFIERS,
+                COMMON_TMP_SPECIFIERS,
+                {'h', specifier_user_home, NULL},
+                {'u', specifier_user_name, NULL},
```

Comment:
specifier_user_name() needs to take scope.

Reply:
> If you want them, then why not for also for user id, group name, and so on?

Ah, because adding the user name specifier here was an afterthought. In the new version I threw in the COMMON_CREDS_SPECIFIERS macro.